.travis.yml

```diff
@@ -36,7 +39,7 @@ script:
 after_success:
   - bash <(curl -s https://codecov.io/bash)
   - .utility/push-javadoc-to-gh-pages.sh
-  - .utility/deploy_snapshot.sh
+  - .utility/deploy.sh
```

Comment:
Can we use `deploy:` instead of `after_success` ?
See: https://docs.travis-ci.com/user/deployment/script/

```yml
deploy:
  provider: script
  script: .utility/deploy.sh
  on:
    branch: master
    java: 'oraclejdk7'
```